ansible-ipi-install/roles/installer/tasks/59_cleanup_bootstrap.yml

```diff
@@ -46,6 +46,7 @@
   find:
```

Comment:
Thanks a lot for this pr

In previous versions it was on libvirt/images, would it make sense to iterate over an array and make it default to the prior and the actual folder so whatever the version being deployed, both are clean?

Reply:
yes totally agreed, I will update